### ISSUE TITLE
What Can & Can't be done with Coral

### ISSUE THREAD

**Issue by ImanHosseini** [google-coral/edgetpu#668]
### Description

There is no clear part in the documentation highlighting the limitations of edgetpu [not low level what ops you can use, but more useful info in terms of: so you want to do CausalLM? BERT? you can't], just to clarify what I infer is this, in case someone is weighting on getting a Coral or looking at issues here to port some model:
1. There are limitations with respect to tensor dimension: https://github.com/google-coral/edgetpu/issues/576#issuecomment-1100499907 , they will not go away and complete info on the limitation 'cannot be disclosed'.
2. Some ops are not supported, and there is no way [for users] to add support for them. No clear timeline or plans for what may later be supported.
3. Taking [1], [2] together, many models will never be able to run on Coral, planning to do GPT* inference on Coral? Don't bother, this will [almost surely] never be do-able. See here: https://coral.ai/docs/edgetpu/models-intro/#model-requirements: 
TLDR: these [limited] supported Ops mean that GPT/ any LM really, is not possible.
A table showing supported (well-known) models in the docs would be nice. 

<details><summary>Click to expand!</summary> 
 
 ### Issue Type

Documentation Feature Request

### Relevant Log Output

_No response_</details>

**Comment by hjonnala** [comment]
Hi, Please check the small selection of model architectures that are compatible with the Edge TPU at: https://coral.ai/docs/edgetpu/benchmarks/

Contributions welcome from the users for any other model architectures that are compatible with the Edge TPU. Thanks!!
